crates/subspace-networking/src/behavior.rs

```diff
@@ -75,7 +75,7 @@ pub(crate) struct Behavior<RecordStore> {
     pub(crate) kademlia: Kademlia<RecordStore>,
     pub(crate) gossipsub: Toggle<Gossipsub>,
     pub(crate) ping: Ping,
-    pub(crate) request_response: RequestResponsesBehaviour,
+    pub(crate) request_response: RequestResponseFactory,
     pub(crate) connection_limits: ConnectionLimitsBehaviour,
     pub(crate) block_list: BlockListBehaviour,
     pub(crate) reserved_peers: ReservedPeersBehaviour,
```

Comment:
I don't like factory name here. Everything was consistently `*Behaviour`, now we have an outlier with `*Factory` :confused: